molecule/testing_user_setup.yaml

```diff
@@ -0,0 +1,51 @@
+---
+- name: Create a non-root sudoer user
+  ansible.builtin.user:
+    name: "{{ item.user }}"
+    shell: /bin/bash
+    group: wheel
+    create_home: yes
+
+- name: Set password for testingUser
+  ansible.builtin.shell: echo {{ item.password }} | passwd --stdin {{ item.user }}
```

Comment:
Manually setting the password, using user.password previously resulted in constant password errors for some reason